src/main/java/com/faforever/client/login/LoginController.java

```diff
@@ -270,6 +306,24 @@ public void onCancelLoginButtonClicked() {
     setShowLoginProgress(false);
   }
 
+  public void onDownloadUpdateButtonClicked() {
+    Platform.runLater(() -> downloadUpdateButton.setOnAction(event -> {
+    }));
```

Comment:
Why callback is empty?

Reply:
Basically I turn off the button that invokes the download. I could also disable the button, but then it looks inactive and would not show the progress so nicely.

Reply:
but remove the PlatFormRunLater... right because this method is called from FXML right?

Reply:
Yes, done